src/pages/proposals/ProposalHistory.vue

```diff
@@ -69,6 +69,8 @@ export default {
       return this.archivedProposals.filter((proposal) => {
         return enabledFilters.some((filter) => {
           return filter.filter(proposal) &&
+            proposal.details_state_s !== 'drafted' &&
```

Comment:
@evgenibir `drafted` and `proposed` should be constants, not strings

Reply:
@Alex-IS fixed